### DISCUSSION TITLE
zh/docs/how-it-works/

### DISCUSSION THREAD

**Discussion by giscus** [casbin/casbin#1466] [Docs comments]
# zh/docs/how-it-works/

Casbin 如何工作

https://casbin.org/zh/docs/how-it-works/

**Comment by MxMurshed** [comment]
The https://github.com/Knetic/govaluate library is not supported anymore! can you add an official replacement?

**Reply by hsluoyz** [reply]
@MxMurshed fixed in commit: https://github.com/casbin/casbin-website-v2/commit/9a0643d5f2ca031b7cc6d866e49e1e47988ea819
